docker-compose.zappdev.yaml

```diff
@@ -0,0 +1,214 @@
+version: '3.9'
+
+services:
+  blockchain_node:
+    image: trufflesuite/ganache-cli:v6.12.2
+    container_name: pointnetwork_blockchain_node
+    healthcheck:
+      test: |
+        wget -qO- http://localhost:$BLOCKCHAIN_PROVIDER_PORT \
+          --header='Content-Type: application/json' \
+          --header="Accept: application/json" \
+          --post-data='{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":0}'
+      interval: 10s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
+    ports:
+      - '$BLOCKCHAIN_PROVIDER_PORT:$BLOCKCHAIN_PROVIDER_PORT'
+    volumes:
+      - blockchain_data:/data:rw
+    environment:
+      - DOCKER=true
+    command:
+      - -v
+      - -p
+      - $BLOCKCHAIN_PROVIDER_PORT
+      - --account
+      - $PK_STORAGE_PROVIDER,0x56bc75e2d63100000
+      - $PK_WEBSITE_OWNER,0x56bc75e2d63100000
+      - $PK_WEBSITE_VISITOR,0x56bc75e2d63100000
+      - -i
+      - $BLOCKCHAIN_NETWORK_ID
+      - --keepAliveTimeout
+      - '20000'
+      - --db
+      - '/data'
+
+  contract_deployer:
+    image: pointnetwork/pointnetwork_deployer:dev
+    container_name: pointnetwork_contract_deployer
+    depends_on:
+      blockchain_node:
+        condition: service_healthy
+    deploy:
+      restart_policy:
+        condition: none
+    volumes:
+      - ./truffle/migrations:/truffle/migrations:ro
+      - ./truffle/contracts:/truffle/contracts:ro
+      - shared_contracts:/build:rw
+      - ./truffle/truffle-config.js:/truffle/truffle-config.js:ro
+    entrypoint: /truffle/run.js
+    environment:
+      BLOCKCHAIN_HOST: $BLOCKCHAIN_HOST
+      BLOCKCHAIN_PORT: $BLOCKCHAIN_PROVIDER_PORT
+      BLOCKCHAIN_NETWORK_ID: $BLOCKCHAIN_NETWORK_ID
+      DEPLOYER_COMPILER_VERSION: native
+      DEPLOYER_BUILD_PATH: /build
+
+  storage_provider:
```

Comment:
We use website_visitor for testing and it has different behavior of website-owner. For example, when testing arlocal download file, after deploying a ZApp website_owner already have the files, so they are loaded from cache. When we use website visitor we can see the files being download from arlocal. Other case when we use website visitor is when we want to test ZApps with more than one identity, we can open two browsers with different identities and use the ZApp, such as sending a email.z from one identity to other one without needing to restart the node.  Storage provider don't seems to being used, I'll remove it. Renaming services, no problem.

Reply:
@tankakatan and @jensendarren one problem that I'm facing to create these two point_node instances is binding the ports which are initialized in point_node docker-compose.yaml. For creating two instances I'll need to remove the port bindings of docker-compose.yaml file, what could broke production env. For now we are with one point node instance and it is all working. If you have any suggestion to solve this issue I can try to implement it.

Reply:
@diogosmendonca ou can create a separate point_node service by copy / pasting the current one in the same compose file and renaming it to website_visitor service and then override the ports as outlined in my earlier comment [here](https://github.com/pointnetwork/pointnetwork/pull/194#discussion_r808738337).

Reply:
Unfortunately docker compose do not allow to override ports. As we had some ports exposed in docker-compose.yaml, I couldn't just extend the service and change the ports. I needed to copy docker-compose.yaml to other file, in case, docker-compose.base.yaml and remove the port declaration of it. In this way was able to expose the ports directly in the containers that uses this base of pont_node configuration, in case, e2e, production and zappdev envs.